windows_containers/index.adoc

```diff
@@ -0,0 +1,33 @@
+[id="windows-container-overview"]
+= Windows container overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for {product-title} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. By using Red Hat subscription, you can get support for Windows Container. For more information see, xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[Release notes].
+
+xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Windows container workloads]:
+For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on Red Hat Enterprise Linux CoreOS (RHCOS) or Red Hat Enterprise Linux (RHEL).
+Windows container workloads support for clusters running on the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+
+xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Enable Windows container workloads]: You need a Machine Config Operator (WMCO) to run Windows workloads in your cluster. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+
+Creating a Windows MachineSet object: You can create a Windows MachineSet object to create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines. You can create a Windows MachineSet object on the following platforms:
+
+* xref:../windows_containers/creating-windows-machinesets/./creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[AWS]
+* xref:../windows_containers/creating-windows-machinesets/./creating-windows-machineset-azure.adoc#creating-windows-machineset-azure[Azure]
+
+xref:../windows_containers/scheduling-windows-workloads.adoc#scheduling-windows-workloads[Windows container workloads]: You can schedule Windows workloads to Windows compute nodes.
```

Comment:
Just to reiterate a previous comment, I would suggest having these separate lines flow together as a paragraph with links combined within.